.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] Kindly add a '/' in front on `node_modules` to specify that it's a directory, and also add `test.md` to your `.gitignore` file